ninja-core/src/main/java/ninja/WrappedContext.java

```diff
@@ -64,12 +64,12 @@ public String getRemoteAddr() {
 
     @Override
     public FlashScope getFlashCookie() {
-        return wrapped.getFlashCookie();
```

Comment:
please leave it that way - it's intentionally referencing the deprecated version.